engine/runtime/src/test/scala/org/enso/interpreter/test/instrument/ExpressionErrorsTest.scala

```diff
@@ -141,16 +174,15 @@ class ExpressionErrorsTest
     val Some(Api.Response(_, Api.InitializedNotification())) = context.receive
   }
 
-  it should "return dataflow errors in method body" in {
+  it should "return panic sentinels in method body" in {
```

Comment:
You should have a test for dataflow errors as well.

Reply:
I added extra tests :+1: